core/src/plugins/filed/python/python-fd.cc

```diff
@@ -138,7 +138,7 @@ static PyThreadState* mainThreadState{nullptr};
 extern "C" {
 #endif
 
-static unsigned long PyVersion()
+static inline unsigned long PyVersion()
```

Comment:
AFAICT that `inline` has no effect here, so I would consider it misleading.

Also - as I learned today - as of C++11 instead of declaring a symbol static you are supposed to put things in an unnamed namespace. (of course it is completely fine to use `static` as that is what is used everywhere)